Extending/Property-Editors/Property-Actions/index.md

```diff
@@ -30,44 +30,40 @@ We use `labelKey` and `labelTokens` to retrieve a localized string that is displ
 
 `isDisabled` is used to disable an Action, which change the visual appearance and prevents interaction. Use this option when an action wouldn't provide any change. In the example above, the action `remove all entries` would not have any impact if there is no entries.
 
-## Getting ready for Property Actions
-A Property Editor needs to be implemented as a Component for it to perform the call to expose its Property Actions.
-The Component must be configured to retrieve an optional reference to `umbProperty`. The requirement must be optional because property-editors are implemented in scenarios where it's not presented.
+## Implementing a Property Action
+The implementation of Property Actions consists of two parts. First a definition of the actions and secondly parsing the lists of actions to `$scope.umbProperty.setPropertyActions().
 
-See the following example:
+Note the check if `umbProperty` exists and is not null or undefined
```

Comment:
Not sure I understand this sentence, @warrenbuckley 🤔 
Does it mean that if `umbProperty` does not exist or is `null` the example will not work?

Reply:
Its just a bit of a safety net - we could probably remove this comment, if you prefer @sofietoft

Reply:
Sure! But, is my understanding of it correct?